frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -699,6 +735,18 @@ mod tests {
 					assert_eq!(balances(&account), (100, 0));
 				}
 			}
+			assert_eq!(
```

Comment:
in line 722, `assert!(MultiPhase::finalize_signed_phase());` can be replaced with `assert_ok!(MultiPhase::elect())` I think, and then the events make a bit more sense.